src/components/DataLoading.js

```diff
@@ -10,7 +11,7 @@ class DataLoading extends React.Component {
     }
     else{
       return (
-          <Spinner animation="border" role="status" style={{float: 'left', marginTop: '50%', marginLeft: '45%', color: 'grey'}}>
+          <Spinner animation="border" role="status" className='wheel-stile'>
```

Comment:
Running this locally - on a laptop with desktop screen (ie not looking at mobile browser) put spinner back at top right corner.
I think this might be because of a typo - `wheel-stile` vs class in css being `wheel-style`

Reply:
Thanks Alexis.
I'll check this and pull again from master to confirm. Let you know when is done. I'll give a look at those errors in the console as well. ;)

Reply:
The wheel is now centred, pulled from master again, it's all ok.
The error messages regarding the <div> tag nested into a <p> tag were already there. This is a react warning that appears only when the app runs locally when is deployed these error messages do not show up. This is due to the React-Bootstrap component `Card.text` that wraps the text into a `<p>` tag.
Can't check if it runs in my mobile as well as the browser because I can't connect to my local network at the moment. On toggle device toolbar looks all ok and as before.